app/Http/Controllers/ArticleController.php

```diff
@@ -19,25 +19,13 @@ public function index(Request $request): AnonymousResourceCollection
         ]);
 
         return ArticleResource::collection(Article::with('user')
+            ->where('status','published')
```

Comment:
you already create builder with isPublished function, you can use it here with `true` parameter